pom.xml

```diff
@@ -214,15 +214,15 @@ THE SOFTWARE.
         <version>${slf4jVersion}</version>
       </dependency>
       <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>log4j-over-slf4j</artifactId>
+        <version>${slf4jVersion}</version>
+      </dependency>
+      <dependency>
         <groupId>commons-logging</groupId>
         <artifactId>commons-logging</artifactId>
         <version>1.1.3</version>
       </dependency>
-      <dependency><!-- [JENKINS-4433] -->
-        <groupId>log4j</groupId>
-        <artifactId>log4j</artifactId>
-        <version>1.2.17</version>
```

Comment:
Added the enforcer rule to the root pom.

I'm thinking that this rule should also be in plugins/pom.xml, so that plugins can't pull in log4j anymore (they should just use slf4j-api). Any thoughts on this?